models/authentication/agent/agent-connected/index.tsx

```diff
@@ -103,24 +94,23 @@ export class AgentConnected {
 
     return {
       ...habilitationInheritedFromOrganisation,
-      agentIsNotVerified,
     };
```

Comment:
```suggestion
return habilitationInheritedFromOrganisation;
```
if we don't need to clone the object